adapters/openx/openx.go

```diff
@@ -140,17 +137,22 @@ func preprocess(imp *openrtb2.Imp, reqExt *openxReqExt) error {
 		imp.BidFloor = openxExt.CustomFloor
 	}
 
+	// copy imp.ext to outgoing request imp.ext, except for prebid fields
 	impExt := openxImpExt{}
-	addImpExt := false
+	if err := json.Unmarshal(imp.Ext, &impExt); err != nil {
+		return &errortypes.BadInput{
+			Message: err.Error(),
+		}
+	}
+	delete(impExt, "bidder")
+	delete(impExt, "prebid")
+	addImpExt := len(impExt) > 0
 
 	if openxExt.CustomParams != nil {
-		impExt.CustomParams = openxExt.CustomParams
-		addImpExt = true
-	}
-	if bidderExt.AuctionEnvironment != openrtb_ext.ServerSideAuction {
-		impExt.AuctionEnvironment = bidderExt.AuctionEnvironment
```

Comment:
don't we want to keep 
```impExt.AuctionEnvironment = bidderExt.AuctionEnvironment``` ?

Reply:
It is now copied as part of `imp.ext.ae`. The [fledge test](https://github.com/prebid/prebid-server/blob/master/adapters/openx/openxtest/exemplary/fledge-no-bids.json) checks for it, although I suppose it couldn't hurt to also add it here